src/main/java/org/opentripplanner/raptor/api/path/PathStringBuilder.java

```diff
@@ -58,6 +59,15 @@ public PathStringBuilder walk(int duration) {
     return start().append("Walk").duration(duration).end();
   }
 
+  public PathStringBuilder vehicleRental(String vehicleId, Duration duration) {
+    return start()
+      .start()
+      .append(" ~ ")
+      .append("Vehicle rent %s".formatted(vehicleId))
+      .duration((int) duration.toSeconds())
+      .end();
+  }
+
```

Comment:
```suggestion
  public PathStringBuilder rental(int duration) {
    return start()
      .append("Rental")
      .space()
      .duration(duration)
      .end();
  }
```
I think using the just "Rental" is enough. Durations in this Raptor utility is `int` in seconds, I think it is good to be consistent. It seems that you are not passing in `vehicleId`, but `vehicleRentalStationId` so I think you should drip the Id. Also, we try to avoid using ids in this Builder, instead we use human readable short information that given the context is enough to identify the ride (Like from, to stop name, mode and time). If you need more info you may use the detailed mode the `path`.